tacs/mpi-compat.h

```diff
@@ -11,4 +11,9 @@ typedef void *PyMPI_MPI_Message;
 #define MPI_Message PyMPI_MPI_Message
 #endif
 
+#if (MPI_VERSION < 4) && !defined(PyMPI_HAVE_MPI_Session)
+typedef void *PyMPI_MPI_Session;
+#define MPI_Session PyMPI_MPI_Session
+#endif
```

Comment:
What does this do?

Reply:
This ensures compatibility for mpi versions older than MPI 4.0 standard (openmpi 3.0 for example), since MPI_Session doesn't exist for those